worlds/witness/options.py

```diff
@@ -151,50 +178,61 @@ class VictoryCondition(Choice):
 
 
 class PuzzleRandomization(Choice):
-    """Puzzles in this randomizer are randomly generated. This setting changes the difficulty/types of puzzles."""
+    """
+    Puzzles in this randomizer are randomly generated. This setting changes the difficulty/types of puzzles.
+    """
     display_name = "Puzzle Randomization"
     option_sigma_normal = 0
     option_sigma_expert = 1
     option_none = 2
 
 
 class MountainLasers(Range):
-    """Sets the amount of lasers required to enter the Mountain.
-    If set to a higher amount than 7, the mountaintop box will be slightly rotated to make it possible to solve without
-    the hatch being opened.
-    This change will also be applied logically to the long solution ("Challenge Lasers" setting)."""
+    """
+    Sets the amount of lasers required to enter the Mountain.

```

Comment:
Number should be used instead of amount here
```suggestion
    Sets the number of lasers required to enter the Mountain.
```